main.go

```
@@ -112,6 +120,12 @@ func main() {
 		smartlogicHealthCacheDuration = time.Duration(time.Minute)
 	}
 
+	smartlogicTimeoutDuration, err := time.ParseDuration(*smartlogicTimeout)
+	if err != nil {
+		log.Warnf("Smartlogic timeout duration %s could not be parsed", *smartlogicTimeout)
```

If the user chooses not to enter timeout value, I'm OK to have some default. However I'm not OK with silently continuing work (because if you deploy in k8s and the service is running you won't check the logs) with a value different than the one the user set.